### DISCUSSION TITLE
How to extend link in the WYSIWG editor via dashboard

### DISCUSSION THREAD

**Discussion by ricardomatias** [directus/directus#10387] [Archive]
I'd like to make all the generated links have the attribute `rel="noopener noreferrer"`, is it possible to do this in the dashboard?

**Comment by rijkvanzanten** [comment]
`noopener` is the default for external (target=_blank) links in TinyMCE. You can add `noreferrer` by overriding the `rel_list` option of TinyMCE (https://www.tiny.cloud/docs/plugins/opensource/link/#rel_list)

**Reply by ricardomatias** [reply]
Thanks! Where can I edit the TinyMCE options?

**Reply by rijkvanzanten** [reply]
There's an "Options Override" JSON input field in the interface options for the WYSIWYG interface that allow you to paste in JSON config that'll be passed to TinyMCEs "init" object on the fly 🙂 

<img width="716" alt="CleanShot 2021-12-13 at 15 23 00@2x" src="https://user-images.githubusercontent.com/9141017/145883064-0fd7aac7-0a51-4a52-b051-b83e3ea25885.png">


In this particular case, you should be able to enter something like:

```json
{
  "rel_list": [
    { "title": "No Referrer", "value": "noreferrer" },
    { "title": "No Opener", "value": "noopener" }
  ]
}
```

**Reply by benhaynes** [reply]
A good one for the Docs... tagged as such.

**Reply by rijkvanzanten** [reply]
@benhaynes We should figure out where in the docs we'd want to put more in-depth examples and details around the various core interfaces 🤔

**Reply by benhaynes** [reply]
I think this would go here: https://docs.directus.io/configuration/project-settings/#branding-style

But we should have a dedicated page for all the Core Interfaces and their setup.

**Reply by rijkvanzanten** [reply]
Hmm I don't think this is a project settings branding configuration, as it's controlling what "rel" attribute value should be added to WYSIWYG generated <a> tags, which might be too specific for that particular page 🤔 

I was thinking we might wanna do an "options reference" for all interfaces

**Reply by ricardomatias** [reply]
Thank you @rijkvanzanten
